templates/census-thank-you.html

```diff
@@ -1,10 +1,15 @@
 {% extends 'layouts/_base.html' %}
 {% from 'components/panel/_macro.njk' import onsPanel %}
-
+{% set page_title = _("Thank you") ~ " - " ~ _("Census 2021") %}
 {% set hide_signout_button = content.hide_signout_button %}
 {% set sign_out_url = content.sign_out_url %}
 
 {% block main %}
+  {% set form = content.form %}
+  {% if form and form.mapped_errors %}
+    {% set error_title = _("There is a problem with this page") %}
+    {% include 'partials/error-panel.html' %}
+  {% endif %}
```

Comment:
When there is an error on the page then the page title needs to be prefixed with `Error:`.
Applies to all other pages.